lifemonitor/api/models/rocrate.py

```diff
@@ -115,6 +114,10 @@ def load_metadata(self) -> dict:
         raise lm_exceptions.NotAuthorizedException(detail=f"Not authorized to download {self.uri}", original_errors=errors)
 
     def download(self, target_path: str) -> str:
+        # report if the workflow is not longer available on the origin server
+        if self._metadata and not check_resource_exists(self.uri):
+            raise lm_exceptions.DownloadException(details=f"Workflow not found on the origin: {self.uri}", status=410)
```

Comment:
`DownloadException.__init__` has an argument called `detail`, without a trailing `s`. This call should not break it since it also has a `**kwargs`, but it's probably failing to do what it's supposed to.

Reply:
Fixed in cf4368a